packages/html-spec/src/forms/forms.json

```diff
@@ -0,0 +1,75 @@
+[
+    {
+        "component": "forms",
+        "selector": [
+            ".k-form"
+        ],
+        "states": [
+            {
+                "name": "form-error",
+                "selector": [
+                    ".k-form-error"
+                ]
+            }
+        ],
+        "options": [
+            {
+                "name": "size",
+                "values": [
+                    {
+                        "name": "small",
+                        "selector": [
+                            ".k-form-sm"
+                        ]
+                    },
+                    {
+                        "name": "medium",
+                        "selector": [
+                            ".k-form-md"
+                        ]
+                    },
+                    {
+                        "name": "large",
+                        "selector": [
+                            ".k-form-lg"
+                        ]
+                    }
+                ]
+            }
+        ],
+        "children": [
+            {
+                "name": "form-field",
```

Comment:
I thought it would be useful to have all of the nested elements, but was not sure how much deeper we can nest. So I have added it as a new component with its children. I am adding the `$ref`, until we decide on how to proceed.